src/renderer/missionWindow/MissionWindow.tsx

```diff
@@ -271,6 +275,54 @@ export default class MissionWindow extends Component<ThemeProps, State> {
         });
         break;
 
+      case 'geofenceTop2':
+        ipc.postUpdateBoundingBoxes(true, {
+          name: 'Geofencing2',
```

Comment:
can we name this box like `Geofencing Keep-In` (or keepout, I dont know what this box is, hence naming needs to change)